src/main/java/io/github/railroad/project/ui/project/newProject/details/ForgeProjectCreationPane.java

```diff
@@ -1,24 +1,38 @@
 package io.github.railroad.project.ui.project.newProject.details;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+
 import io.github.palexdev.materialfx.controls.MFXProgressBar;
 import io.github.palexdev.materialfx.controls.MFXProgressSpinner;
+import io.github.railroad.project.Project;
 import io.github.railroad.project.data.ForgeProjectData;
-import io.github.railroad.task.TaskManager;
+import javafx.application.Platform;
 import javafx.beans.property.ListProperty;
 import javafx.beans.property.SimpleListProperty;
 import javafx.collections.FXCollections;
+import javafx.concurrent.Task;
 import javafx.geometry.Pos;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
 import javafx.scene.control.Label;
 import javafx.scene.layout.BorderPane;
 import javafx.scene.layout.VBox;
 import javafx.scene.paint.Color;
 import org.kordamp.ikonli.fontawesome5.FontAwesomeSolid;
 import org.kordamp.ikonli.javafx.FontIcon;
+import static io.github.railroad.utility.FileHandler.copyUrlToFile;
+import static io.github.railroad.utility.FileHandler.UnZipFile;
+import static io.github.railroad.Railroad.manager;
+import static io.github.railroad.utility.FileHandler.updateKeyValuePairByLine;
```

Comment:
I'm not a fan of using static imports, it makes it hard to see where a method or field is coming from, so please avoid doing this unless its really obvious or necessary.